convene-web/app/views/layouts/_header.html.erb

```diff
@@ -1,9 +1,38 @@
 <header class="header">
   <div class="items-container">
     <h1><%= current_workspace&.name %></h1>
-    <div>
-      <h3>Convene</h3>
-      <h3 class="font-hairline">video meeting spaces</h3>
+    <div class="profile-menu-group" data-controller="menu">
+
+      <%- if Feature.enabled?(:identification) %>
+        <div class="profile-menu">
+          <figure class="avatar">
+            <%= image_tag current_person.avatar_url, size: 30 %>
+            <figcaption><%= current_person.name %></figcaption>
+          </figure>
+
+          <%- if current_person.is_a? Person %>
+            <ul class="profile-menu-dropdown hidden" data-target="menu.items">
```

Comment:
This `ul` probably wants to be a `nav`, and since we only have one element we may not need a `ul` as well. See: https://www.w3.org/TR/html51/sections.html#the-nav-element